### ISSUE TITLE
Call for contributors to define Open Collective v3

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#1745]
Our goal is to bring us back to our initial vision: create a new generation of association that is open, transparent and sustainable. That's why we need to [move beyond the charity framework](https://medium.com/open-collective/moving-beyond-the-charity-framework-b1191c33141) and [make open collectives a new economic unit in the world](https://medium.com/open-collective/from-firms-to-collectives-c139ae27a4ee) that can provide meaningful and sustainable "job opportunities" working for the community.

We want to make sure that we are building things that make sense and that will be useful to the community. To that goal, we are looking for sponsors who would love to experiment with ordering services from collectives (and collectives that would love to give this a try). 

This could be offering support (like [Babel](https://opencollective.com/babel)), bounties (read about [JHipster bounties](https://medium.com/open-collective/jhipsters-bounty-system-and-how-it-saved-the-project-cc118888f642)), workshops, paid talks at conferences, job boards, hiring contributors for specific tasks / projects, etc. You tell us what is top of your mind and needs (and what you have a budget for).

To help you visualize this, here are some examples of what could be possible:

**As a company**
- You want to make sure that the issues and pull requests submitted by your team are being highlighted/prioritized (imagine a special label issues/PR submitted by sponsors)
- You want to prioritize a particular pending issue (bounty)
- You want to hire remotely talented people that you know are supported by the community
- You want to get someone who is a verified contributor to an open source project to come to your office to make a presentation about the project
- You want to hire the equivalent of in the old days a "Certified Microsoft Developer" but "certified" by the community.

**As a contributor to a collective**
- You are a contributor to an open source community and you are available for hire, you'd love to be able to "advertise" that
- Maybe you are available to hire as a digital nomad and only for limited time (part time or project based)
- Maybe you just want to pick particular problems / issues that you'd love to work on that some companies are willing to pay for
- Maybe you love doing conferences and present the project, you'd love to make a living out of it (at least part time)
- Maybe you love teaching and you'd love to offer workshops

Does this make sense to you? What would you be willing to do to help your community be more sustainable? Tell us more.

## Who is this new version of Open Collective for?

Our primary users so far have been open source communities. With this new version we want to particularly target the communities that want to be financially sustainable and companies that want to support Open Source because they already understand that [it makes total business sense](https://fosdem.org/2019/schedule/event/community_towards_sustainable_solution_open_source_sustainability/) (it enables them to attract and retain the best talent, it makes sure their team can tap into the wisdom and experience of a broader community to get the job done, better and faster).

Beyond the tech community, we also believe that we need to bring the open source values and ways of working together to the real world. 

Our generation is facing a big global challenge. We urgently need to move away from fossil fuel and build a much more sustainable society that is respectful of the environment and of our communities.

We all have a role to play in this fight and we are no exception. That's why we also want to build this new version of Open Collective with local communities in mind. 

P.S: If you are curious, here is a first mockup of what this new version of open collective could look like: https://github.com/opencollective/opencollective/issues/1744

**Comment by piamancini** [comment]
We've been working on an MVP for this. This are the persona assumptions: 

### Services MVP - Personas 

Bob is a contributor to webpack and wants to help fund the community  
Bob is a contributor to webpack and wants to make money with my webpack skills working on what he loves. 

Sean is a core contributor to webpack and wants more people to work on webpack
Sean wants people who can fix issues / do features
Sean wants people to write documentation
Seans wants people to help with fundraising. 
Sean is a core contributor to webpack and he wants to help fund the community 
	
Alice is the CTO of Acme and wants webpack to work well in her product 
Alice needs someone to explain webpack to the team 
Alice wants an immediate issues to be solved/fixed/explained 
Alice wants to hire engineers who are very familiar with webpack  

### MVP Whiteboard 
![img_5741](https://user-images.githubusercontent.com/3671070/53011983-a525aa00-3439-11e9-9059-156223a7ecc7.jpg)

### On the request side: 
Alice is looking for someone to fix/build a feature she needs on webpack 
Posts an issue with the request & a pledge (bounty)
Sean tags Bob and if Bob is interested assigns him the issue
Request is fulfilled
Alice pays the pending order 
Bob submits an expense to the collective. 

### On the offer side 
(Things are fuzzier here)

A) Sean creates the tiers for the collective (become a sponsor & become a backer style)
Bob can also create a tier: I am offering to do bug fixes / support for $xx (and can say if they'll be able to give back something to the community)
Alice sees the notice, contacts Bob directly, pays him directly
Bob makes a donation to webpack after being paid 

B) Acme posts a service (we do great websites and we give back to webpack xx% of all our contracts (or business driven through here?) We are good citizens and that's an extra reason to hire us. 
LOLCorp sees Acme's post and hires them (through their own channels, website etc)
Acme makes a contribution to webpack from that %.

**Comment by ghuntley** [comment]
> You want to make sure that the issues and pull requests submitted by your team are being highlighted/prioritized (imagine a special label issues/PR submitted by sponsors)

I'm keen to see this impl with the current evolution of OpenCollect. My preference would be OpenCollect integration with Zapier and then letting folks like myself wire up the world. https://github.com/opencollective/opencollective/issues/464#issuecomment-410693144

**Comment by ghuntley** [comment]
As to the future. Look to the past, specifically the webpack 400k interview:

> There’s a lot of companies out there that will hook you up if you give them some exposure in your docs or README. You’ve got to give them some business value, but the hosting companies and cloud providers are always trying to one-up each other. If they’re getting their logo on the README of a project with 22,000 stars, drop the mic.
>
> Sean’s response:
>
> You’re so right it’s ridiculous. By being a backer or sponsor, you instantly get to have your mark on not only our docs and README, but if you go to our website and scroll down you see our segments of gold sponsors and silver sponsors. Every single backer we have from Open Collective shows on our docs page.
> 
> It just reinforces that every single one of these contributions is so important. Whether it’s the CEO of Stripe, who secretly is on there giving $10, to Capital One, who was our first ever sponsor. Even AG Grid, another open source project, is there. I started talking with them at a conference in Copenhagen and said “Hey, you’re trying to build your enterprise and we have lots of hits on our documentation.

Specifically

> Instead of spending $20,000 to sponsor one conference, you could spend the same for millions of impressions by being on our docs page.” They tell me they get 50–100 clickthroughs daily. It’s easy marketing, more impactful than conference sponsorships.

**Comment by ghuntley** [comment]
As for:

> You are a contributor to an open source community and you are available for hire, you'd love to be able to "advertise" that

This is a big issue ATM. Over at reactiveui.net/support we have revenue but not enough to full sustainability to hire maintainers in western civilization. Whilst this condition is true we need to provide ROI to our backers. Specifically this looks like hiring a tech writer. The problem is....

How do we find a tech writer? I've got my connections via twitter and working through options ATM but this problem of having money, needing ROI, unable to find good options is real.

**Comment by ghuntley** [comment]
> You want to hire remotely talented people that you know are supported by the community

ReactiveUI has started paying one of its maintainers who lives in eastern europe. We have been paying the person in recent times because of the previously mentioned problem of ROI. For far in the last month our website has dramatically improved and backers are seeing a visible ROI.

**Comment by ghuntley** [comment]
> You want to prioritize a particular pending issue (bounty)

IBM does this to get folks to port open-source software to the IBM power and Z (mainframe) platforms. See

https://www.bountysource.com/issues/70049300-powerpc-enable-pow-powf-in-libmvec-vector-math-library-on-power-vsx

**Comment by ghuntley** [comment]
> Maybe you love doing conferences and present the project, you'd love to make a living out of it (at least part time)

I think there's a place in the future for open-source projects to have their own paid developer advacados. One avacado time sliced between many different collectives. 

See today's announcement in the Haskell community.  

https://www.reddit.com/r/haskell/comments/as01d4/fp_complete_is_hiring_a_technical_communicator/

**Comment by ghuntley** [comment]
> You want to hire remotely talented people that you know are supported by the community

Hmm. I think production of case studies that proves that backing an open-source project has a direct correlation to attracting quality candidates and a way to fill hiring pipelines is what's needed ATM. Like we all kinda "know this" but let's prove it. Then people at companies can point their managers/executive teams to this material.

TLDR: don't do anything but produce marketing case studies.

**Comment by ghuntley** [comment]
> You want to get someone who is a verified contributor to an open source project to come to your office to make a presentation about the project

I've personally been experimenting with doing virtual meetups. Like dialling into meetups all around the globe from Sydney, Australia. It doesn't matter which timezone the audience is in. Just doing it.

Feedback so far has been good. It might be an idea (TM) to ask people at the Meetup to pay it forward and fund either my time directly or the open collective that I'm representing.

**Comment by piamancini** [comment]
@ghuntley 
> I'm keen to see this impl with the current evolution of OpenCollect

yes, we'll likely do this with Github Labels first (OC as a Github App). But feel free to poke around for a zapier integration we'll be happy to help you do it. 

> How do we find a tech writer?
 
this is the goal of the offer part to enable tech writers to offer their services - But yes discovery is key and we shouldn't underestimate the problem.

wouldn't ROI for your backers be better/more fixes, people responding to their bounties, giving support?

> IBM does this to get folks to port open-source software to the IBM power and Z (mainframe) platforms. See

The difference in this case would be that IBM would be funding the community and then someone from the community (a contributor) will file an expense to the community. The result is sustainability of a community as well as a relationship that IBM has with an existing legal entity (the open source collective or any other host) instead of a one off to an individual. 


> virtual meetups

this is a great idea.

**Comment by slavingia** [comment]
very cool. excited for this! maybe Gumroad could be a customer...

**Comment by BenJam** [comment]
All this is awesome 🙌 some rando quick ideas to take or leave:

I would recommend pitching the use cases in the context of projects with 'disposable' income (like Webpack) _and_ projects that are yet to get there (like Octobox). You're likely to get two different set of characteristics there (push money out and pull money in basically). 

There's also an argument to suggest that even v3 (as pitched) is 'playing in the long tail' rather than focussing on the (comparative) whales, as the pull projects are more valuable in aggregate.

**Comment by piamancini** [comment]
@BenJam that is great feedback. we'll do the exercise from a smaller project's perspective.

**Comment by BenJam** [comment]
I'm not sure whether my anti-compete clause with Tidelift would stop me from doing so... Paging @tieguy.

**Comment by piamancini** [comment]
@BenJam any chance you can help us think thorough this from a smaller collective's point of view? 
What would you like to see? I'm sure the non compete doesn't cover commenting on an open issue :)

**Comment by BenJam** [comment]
Alright, quick thoughts and all IMO:

We (Octobox) are a little weird because our goal, ultimately, is to detach how we make our money from how we spend our time, to pay for 2-4 'FTE' living comfortably in the 'western world' (as contrived a saying as that is) and to have them focus their time working on Octobox as a product. Because products scale, and people's time doesn't. 

I would say that assumption you have in the OP is a model in which people are predominantly transacting based on the value of their time. But like I say, Octobox is weird and most open source projects will use a consulting model.  

The way I see it, in this model open source projects operate as a kind of open agency in which the project acts like a brand umbrella, bringing customers to a single storefront to purchase services and giving everyone within that community a halo effect (on the value of their time) as an official contributor. In return for that halo effect they (morally as stated, but easily legally backed up) give a %age of the value they get from the work through the project back to the community. If everyone's successful then the project can use that %age to reinvest in work that lacks other incentives to do (generally the boring stuff). 

So IMO the goal on the smaller projects side is to do everything to get them to the point that that have enough revenue to reinvest in themselves. All of which is based on another assumption: that they can get there. 

Projects aren't going to be successful at this approach unless they can connect with their monied user base. Whether this is a large user base willing to pay a small amount or a small user base with some niche use cases that require more support, the project still needs to find those companies and/or the contributors to help them. So maybe the #1 thing that you can help smaller projects with is to find and connect with their target market, and visa versa.

Once that ball is rolling you need a place to list the services available, build that social proof (clients, users, contributor profiles etc) and provide clear routes into the community for typical requests.  

One thing I noticed that's relevant for projects big and small: as written above it looks like a lot of the admin (billing, invoicing, estimating etc.) is the responsibility of the contributor. But I wonder whether, in exchange for some support services, the project could streamline some of the costs involved here while ensuring that those %age revenues come back to the project. template agreements for paying customers, members of the community, professional indemnity etc.

**Comment by piamancini** [comment]
_Projects aren't going to be successful at this approach unless they can connect with their monied user base_
I think this is key here and we have overlooked it in the initial design / thinking of v3. (cc / @xdamman)

 _So maybe the #1 thing that you can help smaller projects with is to find and connect with their target market, and visa versa._
Great feedback and yes, couldn't agree more. 

_But I wonder whether, in exchange for some support services, the project could streamline some of the costs involved here while ensuring that those %age revenues come back to the project. template agreements for paying customers, members of the community, professional indemnity etc._

Yes, we've been talking about this. The gist is having a Pro version of OC that can be purchased by individuals (as oppose to collectives, with the potential of a collective paying for it and making it available to all it's contributors). I imagine a service that for example can enable email based invoices for something for $7 a month, that Andrew (and others) would use for it's monthly heroku bills

**Comment by BenJam** [comment]
> PS: the octobox badge is awesome! (cc @znarf this is what we want for OC backers/sponsors via the bot!)

Viral marketing FTW!

**Comment by znarf** [comment]
@piamancini @BenJam which badge are we speaking about? how can that be replicated for the opencollective bot?

**Comment by BenJam** [comment]
I think @piamancini is referring to the Octobox badge next to her avatar (and my own ) which is added when interactions occur from a user through a third-party app like Octobox.

**Comment by znarf** [comment]
@piamancini I don't think this is relevant for the bot today, because the bot is posting messages itself, not on behalf of users. Maybe for other use cases?

**Comment by mattwynne** [comment]
I agree with @BenJam. Our goal at Cucumber Ltd is also to get Annual Recurring Revenue so that we can relax and prioritise what matters rather than what pays the bills. We're doing that by investing in a premium / commercial product within the company but I would love to see some thinking here about how OpenCollective could provide collectives with that kind of stability.

Thinking purely from the perspective of an open-source project, I think the main opportunity we see from Alice's point of view is twofold:

1) I want to be associated with your brand to increase the prestige of my engineering organisation, help me hire, get hits to our website etc.
2) I have a "pet feature" that is unlikely to get prioritised the the community but that I would be willing to pay for.

For (2) there might be several Alices at different companies who would want to chip in on a "pet feature". This reminds me a bit of the way https://cobudget.co/ from https://github.com/enspiral works.

**Comment by piamancini** [comment]
pinging @alannallama for the cobudget mention.

**Comment by BenJam** [comment]
> ad badge to their profile anytime they open an issue, right?

That is an interesting, non-intrusive way of signalling that someone is a ‘patron’ of the project. I like it. 

It would require that you build quite a lot of software just to support it though; unless you’re already intending to go down that route.

**Comment by alannallama** [comment]
@mattwynne I actually originally created [Cobudget](https://cobudget.co), so it's awesome to have it mentioned in this context 😄

Open Collective does a great job focusing on bringing money in, but there are opportunities to help Collectives with tools for participatory financial decision making about where money is spent, too. As you say, if we could surface available resources and capacity to build stuff and connect them together, while helping both financial and time contributors feel empowered by the interaction, that would be amazing. Cobudget or a Cobudget-like set of features could be helpful for that. 

I'm keen to talk more about it if people are interested! I think it meshes very well with bounties and the ideas the team has for V3.

**Comment by gusaus** [comment]
I think improving the seach and discovery would make it easier for foundations, nonprofits and other organizations to commit funding and resources.  

* support campaigns that align with their mission or target vertical
* support open source projects and commununties they benefit from
* provide training or mentorship
* sponsor a meetup, hackathon, or working group that aligns with their mission or target vertical

Ideally supporters could filter their search on things like - 
* Geographic area - projects or groups in my area
* Category or vertical - Tech, Education, Nonprofit/NGO, Media/Publishing, Music, Community/Civic Participation, Social Impact, etc.
* Project/collective type - Open source project, Meetup, Cooperative, etc. 
* Host type - nonprofit, for profit, etc.
* Projects looking for talent - possibly filter on skill needs or demographics
* Funding level - fully funded, projects with no funds, matching offers, etc.
* Matching offers

https://www.donorschoose.org/donors/search.html provides some good inspiration for search and discovery

**Comment by piamancini** [comment]
@BenJam @mattwynne Following your suggestions here, we started brainstorming about ways to improve the collectives' ability to reach out to their users (potential sponsors/backers) This is where our thinking is atm https://github.com/opencollective/opencollective/issues/1869

**Comment by BenJam** [comment]
🙌

**Comment by BenJam** [comment]
new comment
